constraints.txt

```diff
@@ -8,7 +8,7 @@ chardet==3.0.4
 click==6.7
 clickclick==1.2.2
 connexion==2.6.0
-cryptography==2.6.1
+cryptography==41.0.2
```

Comment:
Well, we don't use it directly. It's installed because other packages require it. I can say for sure that these other packages don't have any conflict with this version in the sense that this version meets their own requirements/constraints. But I cannot say for sure that the packages we include have their dependencies accurately specified.

I will say that on the Snyk page for this particular vulnerability, its suggested resolution is just to update the cryptography version and not to update the versions of the packages that require it. Again, though, I wouldn't consider that to be a guarantee of compatibility.

When the same vulnerability was found in the cfs-operator repo and required a similar version jump, Ryan's opinion was that this wouldn't present a compatibility issue, but that was opinion, not based on testing.

I do think the version number jump is more dramatic than it seems in some senses -- this package appears to bump its major number with very little prompting.

I'm guessing that this package is included because of some underlying authentication that is happening, or possible because of password hashing inside of some package. I'll see if I can deploy the updated container on mug or some place to see if there are any problems.

Reply:
@jsollom-hpe I have updated the PR description with the results of my testing. Please let me know what you think.